### DISCUSSION TITLE
Bug? Issue description in the backlog has disappeared

### DISCUSSION THREAD

**Discussion by Andrea-MW** [community/community#143545] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Dear community,

I had a quite complex backlog item in the backlog (a draft, not yet an issue) for which the description has just disappeared. The item is shown with its title in the backlog list but when I click on it, I see just an empty screen, even the title is emtpy. 

Do you have an idea how can I recover the backlog item? Or ist this a bug?

![image](https://github.com/user-attachments/assets/0f90df79-823a-40b9-b7bd-eeac10ffa991)
![image](https://github.com/user-attachments/assets/8aaf7075-5660-4986-a966-fcbb4df5c549)

Thanks for your support in advance
Andrea

**Comment by thomassiard** [comment, marked as answer]
Here are a few steps you might try to recover it:

Refresh the Page or Clear Cache: Sometimes, display issues can happen due to a browser cache problem. Try refreshing the page, or clearing the cache to see if it resolves the issue.

Check for Autosave Versions: Some platforms autosave drafts at intervals, so check if there's an option to access a previous version or history of the backlog item.

Switch Browsers or Devices: Occasionally, a different browser or device can show content that seems "missing" on another due to compatibility quirks.

Report the Bug: If none of these work, it’s possible that this is a bug affecting the draft handling or display. Reporting it to the support team or filing it as a bug can help them investigate, and they may be able to recover your item from a backup or database state.

**Reply by Andrea-MW** [reply]
Clearing the browser cache helped. Thanks a lot for your hint!! :-)

**Comment by coyamat** [comment]
+1
